ui/app/src/components/RejectDialog/RejectDialogContainer.tsx

```diff
@@ -29,12 +29,16 @@ import { updateRejectDialog } from '../../state/actions/dialogs';
 import { showErrorDialog } from '../../state/reducers/dialogs/error';
 import { batchActions } from '../../state/actions/misc';
 
+const typeCustomReason = 'typeCustomReason';
 export function RejectDialogContainer(props: RejectDialogContainerProps) {
   const { items, onClose, onRejectSuccess, isSubmitting } = props;
   const [checkedItems, setCheckedItems] = useState([]);
-  const [rejectionReason, setRejectionReason] = useState('');
+  const [rejectionReason, setRejectionReason] = useState(typeCustomReason);
   const [rejectionComment, setRejectionComment] = useState('');
   const [rejectionCommentDirty, setRejectionCommentDirty] = useState(false);
+  const isSubmitDisabled = useMemo(() => {
+    return checkedItems.length === 0 || rejectionComment.trim() === '' || isSubmitting;
+  }, [checkedItems, rejectionComment, isSubmitting]);
```

Comment:
Too short a computation to be worth a memo.
```suggestion
  const isSubmitDisabled = checkedItems.length === 0 || rejectionComment.trim() === '' || isSubmitting;
```